Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -6546,329 +10487,346 @@ def geo_exception_list_command(client: Client, args: dict[str, Any]) -> CommandR
+
+    Returns:
+        Tuple[List[Dict[str, Any]], str, List[Dict[str, Any]]]: Filtered output to xsoar,
+        pagination message and response output.
+    """
+    if isinstance(client, ClientV2) and isinstance(response, dict):
+        response = response["results"]
+    if internal_path:
+        response = dict_safe_get(response, internal_path)
+    elif not isinstance(response, list):
+        response = [response]
+    if sub_object_id and isinstance(response, list):
+        group_dict = find_dict_in_array(response, sub_object_key, sub_object_id)
+        response = [group_dict] if group_dict else []
+        if not response:
+            raise DemistoException(ErrorMessage.NOT_EXIST.value)
+    if name_filter := args.get("search") and isinstance(response, list):
+        for obj in response[:]:
+            if (name := obj.get("name")) and name_filter not in name:
+                response.remove(obj)
+    response, pagination_message = paginate_results(client.version, response, args)
+    parsed_data = parser_handler(response, data_parser)
+    return parsed_data, pagination_message, response
+
+
+def parser_handler(
+    data: List[dict[str, Any]], data_parser: Callable
+) -> List[dict[str, Any]]:
+    """Parse the data by parser command.
+
+    Args:
+        data (List[Dict[str, Any]]): The data to parse.
+        data_parser (Callable): Parser command.
+
+    Returns:
+        List[Dict[str, Any]]: Filtered output to xsoar.
+    """
+    return [data_parser(data_to_parse) for data_to_parse in data]
+
+
+def paginate_results(
+    version: str, response: Union[List, dict[str, Any]], args: dict[str, Any]
+) -> tuple[list, str]:
+    """Executing Manual paginate_results  (using the page and page size arguments)
+        or Automatic paginate_results  (display a number of total results).
+    Args:
+        response (Union[Dict[str, Any], List[Dict[str, Any]]]): API response.
+        page (int, optional): Page number of paginated results.
+        page_size (int, optional): Number of ip-list per page.
+        limit (int, optional): The maximum number of records to retrieve.
+    Returns:
+        Tuple[dict,str]: Output and paginate_results  message for Command Results.
+    """
+    response = response if isinstance(response, list) else [response]
+    page = arg_to_number(args.get("page"))
+    page_size = arg_to_number(args.get("page_size"))
+    limit = arg_to_number(args.get("limit", LIMIT_SIZE))
+
+    output = response
+
+    if limit:
+        output = response[:limit]
+        pagination_message = f"Showing {len(output)} rows out of {len(response)}."
+    if page and page_size:
+        if page_size < len(output):
+            first_item = page_size * (page - 1)
+            return output[first_item: (first_item + page_size)], pagination_message
+
+        pagination_message = f"Showing page {page}. \n Current page size: {page_size}"
+        return output[:page_size], pagination_message
```

Comment:
if page_size > len(outputs)  then outputs == outputs[:page_size]
for example if x = [1,2,3] then x[:4] == [1,2,3] == x

hence:

```suggestion
        return output, pagination_message
```